src/shared-components/callout/style.ts

```diff
@@ -37,8 +40,27 @@ const Icon = styled.div<{
   }
 `;
 
+const CrossIconContainer = styled.a`
+  align-items: center;
+  border-radius: 50%;
```

Comment:
Yeah, I debated that for a bit, but ended up going with the `a` tag. Since you're recommending a button though, we probably should do that. Lemme change it. Will re-request a review once it's changed and all set with the checks. Thanks.

Reply:
Sounds good! There was an accessibility training session we did a few months ago that helped define where each tag should be used. I'll see if I can find it

Reply:
That would be helpful. Thanks a lot.